externs/lcevc.js

```diff
@@ -16,58 +16,47 @@ var LcevcDil = {};
 LcevcDil.LcevcDIL = class {
   /**
      *  @param {HTMLVideoElement} media
-     *  @param {HTMLCanvasElement} canvas
-     *  @param {LcevcDil.LcevcDilConfig} dilConfig
+     *  @param {Element} canvas
```

Comment:
Shouldn't this be a canvas element specifically?

Reply:
(good catch, really ought to be as it can't be anything else)